core/Plugin/Manager.php

```diff
@@ -1184,6 +1184,11 @@ public function loadPlugin($pluginName)
         if (isset($this->loadedPlugins[$pluginName])) {
             return $this->loadedPlugins[$pluginName];
         }
+
+        if (!$this->isPluginInFilesystem($pluginName)) {
```

Comment:
@diosmosis I think it might be good if we undo this particular change just to not break any possible use cases or cause any side effects. We can always add it later if needed for some reason. I just want to prevent adding any possible side effects or causing any issues when there isn't really a problem (even the original issue itself is very minimal and very rarely any issue etc)

Reply:
:+1: this is what I was thinking at first. However, I figured if there's no plugin folder, then we wouldn't want it to work. I can revert it though.

Reply:
👍  I can't really think of much of an issue either but you never know. For starters it may slow down Matomo quite a bit as it would read the filesystem for every plugin (of which there can be many). This can make Matomo load quite slow and depending on the disk cause bottlenecks etc

Reply:
@tsteur updated